### PR TITLE
Add ABR setup information to Site Health

### PR DESCRIPTION
**Pull request by nfmohit**
## Summary

Addresses issue:

- #7290 

## Relevant technical choices

This PR adds ABR specific information to Site Health.

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: includes/Modules/AdSense.php [hunk 1]
```diff
@@ -202,36 +202,80 @@ public function on_deactivation() {
...
 				'label' => __( 'Web Stories Ad Unit ID', 'google-site-kit' ),
 				'value' => $settings['webStoriesAdUnit'],
 				'debug' => $settings['webStoriesAdUnit'],
 			),
+			'adsense_setup_completed_timestamp' => array(
+				'label' => __( 'AdSense setup completed at', 'google-site-kit' ),
+				'value' => date_i18n(
+					get_option( 'date_format' ),
+					$settings['setupCompletedTimestamp']
+				),
```
Comment (hussain-t) [inline includes/Modules/AdSense.php:239]: According to the AC, if the setting is empty, the value should be `Not available`.

>* Value: date formatted according to the site's locale. If the setting is empty, it should be `Not available`

**Review by hussain-t — CHANGES_REQUESTED** [review]
Thanks, @nfmohit. Your code looks great in general. However, one detail from the AC still needs to be fully addressed. For more context, refer to my comment. Once that is addressed, we should be good to go.

File: includes/Modules/AdSense.php (hunk 1 above)
Comment (nfmohit) [reply includes/Modules/AdSense.php]: Addressed, thank you!

**Comment by nfmohit** [comment]
> Thanks, @nfmohit. Your code looks great in general. However, one detail from the AC still needs to be fully addressed. For more context, refer to my comment. Once that is addressed, we should be good to go.

Excellent catch, updated, thank you!

**Review by eugene-manuilov — APPROVED** [review]

**Pull request merged**